src/app/invest/invest.component.html

```diff
@@ -1,71 +1,83 @@
 <div *ngIf="{
     wallet: wallet$ | async,
     project: project$ | async
-} as states" class="container root-content-container">
+} as states" class="container">
     <ng-container *ngIf="states.wallet !== null && states.project !== null else spinner">
         <div class="row">
-            <p class="col-12 mt-3 mb-lg-4 ml-lg-3 page-title text-uppercase">
+            <div class="app-page-title-back" (click)="backToSingleOfferScreen(states.project.uuid)">
                 <i class="fas fa-long-arrow-alt-left mr-3"></i>
                 Invest in
-            </p>
-            <h2 class="col-12 mb-3 mt-3 text-uppercase project-title">
-                {{ states.project.name }}
-                <div class="dropdown-divider mt-lg-3"></div>
-            </h2>
-
-            <div class="col-12 d-lg-flex align-items-center">
-                <div class="col-lg-6">
-                    <div class="invest-text text-uppercase">
-                        <p>I want to invest</p>
+            </div>
+            <div class="invest col-12">
+                <h1 class="app-basic-title">{{ states.project.name }}</h1>
+                <div class="dropdown-divider"></div>
 
-                        <app-money-input-field
-                                placeholder="Enter invest amount"
-                                [control]="investForm.get('amount')">
-                        </app-money-input-field>
+                <div class="invest__content">
+                    <div class="invest__text-left col-lg-6">
+                        <div class="invest__input-label">
+                            <p>I want to invest</p>
+                            <app-money-input-field
+                                    placeholder="Enter invest amount"
+                                    [control]="investForm.get('amount')">
+                            </app-money-input-field>
+                        </div>
+    
+                        <div class="invest__range-labels">
+                            <p>{{ states.project.min_per_user | currencyDefault }} Min investment</p>
+                            <p>{{ states.project.max_per_user | currencyDefault }} Max investment</p>
```

Comment:
This code is not assured that `states.project` exists, so it will throw an error that `min_per_user` does not exist on null object. Wrap these cases in ngIf='!!states.project else spinner'. Look on other places in app where this spinner is used.